automated install/basic-install.sh

```diff
@@ -1052,7 +1052,7 @@ installLogrotate() {
   # the local properties of the /var/log directory
   logusergroup="$(stat -c '%U %G' /var/log)"
   if [[ ! -z $logusergroup ]]; then
-    sed -i "s/# su #/su ${logusergroup}/" /etc/pihole/logrotate
+    sed -i "s/# su #/su ${logusergroup}/g;" /etc/pihole/logrotate
```

Comment:
Adding `global` here since there are now two places in `advanced/logrotate` where this has to be applied